.github/workflows/continuous-integration.yml

```
@@ -47,6 +47,7 @@ jobs:
           php-version: "latest"
           tools: "composer:${{ matrix.composer-version }}"
           coverage: "none"
+          fail-fast: true
```

Just realized I implemented this incorrectly, it should be:
```suggestion
        env:
          fail-fast: true
```

Fixed in an updated commit.